### DISCUSSION TITLE
网页版强制改成新版页面了，回不去旧版了

### DISCUSSION THREAD

**Discussion by Yia-wooh** [the1812/Bilibili-Evolved#4228] [综合讨论 / General]
插件里好多首页功能直接就没用了

**Comment by Geeyun-JY3** [comment]
https://github.com/the1812/Bilibili-Evolved/discussions/4206#discussioncomment-5938148

**Reply by GalaxySnail** [reply]
不起作用。设置 `go_old_video` 等 cookie 已经不管用了

**Reply by Yia-wooh** [reply]
确实，我刚安上没有用

**Reply by Geeyun-JY3** [reply]
不管有没有用，Bilibili Evolved 一直都没打算过做返回旧版页面的功能。
相关 discussion 打的 label 一直是 `feat: rejected`。

**Comment by Geeyun-JY3** [comment]
至于`清爽首页`和`极简首页`等组件不适配新版首页的问题，请参见：<https://github.com/the1812/Bilibili-Evolved/issues/3786#issuecomment-1304385603>、#3639、#3981、#3720
